src/toil_lib/validators.py

```diff
@@ -0,0 +1,29 @@
+import os
+from subprocess import Popen, PIPE
+
+from toil_lib import require
+
+
+def bam_quickcheck(bam_path):
+    """
+    Perform a quick check on a BAM via `samtools quickcheck`.
+    This will detect obvious BAM errors such as truncation.
+
+    :param str bam_path: path to BAM file to checked
+
+    :rtype: boolean
+    :return: True if the BAM is valid
+    """
+    directory, bam_name = os.path.split(bam_path)
+    p = Popen(['docker', 'run', '-v', directory + ':/data',
+               'quay.io/ucsc_cgl/samtools:1.3--256539928ea162949d8a65ca5c79a72ef557ce7c',
+               'quickcheck', '-vv', '/data/' + bam_name], stdout=PIPE, stderr=PIPE)
```

Comment:
Capture neither stdout nor stderr. Toil already captures both and redirects them to a file. Whatever samtools prints will automatically appear in that file. IOW, `subprocess.call` would be right tool in this scenario.